### PR TITLE
Resolved Issue#16

### LINKED ISSUE #16: Update Date Format to Reflect en-IN Format
Currently, dates in the application are displayed in the `en-US` format (MM/DD/YYYY). It would be better to:

1. Change the default date format to `en-IN` (DD/MM/YYYY).
2. **Bonus:** Implement functionality to automatically detect the user's locality (e.g., by using services like [ip-api.com](http://ip-api.com/) or similar) and display dates in the appropriate local format.

### PR DESCRIPTION
**Pull request by MadhavDhall**
### Related Issue  
Closes #16 

### Type of Change
Put `x` inside the square bracket to specify what type of change your PR is:  
- [x] New Feature  
- [ ] Bug Fix  
- [ ] Code Refactor  
- [ ] Documentation Update  
- [ ] Other (please specify): 

### Description of Change  
Changed the default date format to en-IN (DD/MM/YYYY).
Implemented functionality to automatically detect the user's locality and display dates in the appropriate local format.

### Implementation Details  
Changed the default date format to en-IN (DD/MM/YYYY).
Implemented functionality to automatically detect the user's locality and display dates in the appropriate local format. 
User timezone is fetched by Intl.DateTimeFormat().resolvedOptions().timeZone and added in options of date.toLocaleString()

### Demo  
Screenshot 2024-12-26 at 10 20 58 PM
At Indian TimeZone

WhatsApp Image 2024-12-26 at 22 11 55

At Australian TimeZone

### CODE REVIEW

File: app/(components)/TicketCard.jsx [hunk 1]
```diff
@@ -11,10 +11,11 @@ const TicketCard = ({ ticket }) => {
       hour: "2-digit",
       minute: "2-digit",
       hour12: true,
+      timeZone: Intl.DateTimeFormat().resolvedOptions().timeZone
     };
 
     const date = new Date(timestamp);
-    const formattedDate = date.toLocaleString("en-US", options);
+    const formattedDate = date.toLocaleString("en-IN", options);
```
Comment (AyushDharDubey) [inline app/(components)/TicketCard.jsx:18]: why not fetch the locale string "en-IN" also form Intl

Comment (MadhavDhall) [reply app/(components)/TicketCard.jsx:18]: Can be done, in issue it was specifically mentioned en-IN, hence i specifically made it for en-IN.
Shoul i change it now??

Comment (AyushDharDubey) [reply app/(components)/TicketCard.jsx:18]: yes please do

**Comment by MadhavDhall** [comment]
Hello @AyushDharDubey sir
As said, I have added userlocale form Intl. Now the userlocale changes according to user system's settings.

Screenshot 2024-12-27 at 12 49 02 PM
In laptop with en-US setting

WhatsApp Image 2024-12-27 at 12 46 31

In mobile with en-IN setting

**Comment by AyushDharDubey** [comment]
Good work @MadhavDhall,

i can see that the changes are capable of closing the issue
thank you for your contribution

**Pull request merged**